### PR TITLE
Rbotha/feature 85 week 7 account settings spa

### PR DESCRIPTION
**Pull request by ReinhardtBotha**
- Styling added (as close as possible)
- two different menus implemented, one for mobile and one for large screens
- upload and delete profile picture
- show default picture if user has no profile picture
- reset password
- delete account

### CODE REVIEW

**Comment by BryanCaldeira** [comment]
image

I didn't have a user profile pic, default user image should be shown

File: pages/settings/index.html [hunk 1]
```diff
@@ -1,102 +1,130 @@
...
+            <option value="notificationsOtp" class="text-body-1 medium">Notifications</option>
+            <option value="securityOtn" class="text-body-1 medium">Security</option>
+          </select>
+
+          <div class="account-settings__side-menu" id="sideMenu">
+            <a href="#" class="account-settings__menu-btn" id="btn1">Basic Info</a>
+            <a href="#" class="account-settings__menu-btn" id="btn2">Password</a>
+            <a href="#" class="account-settings__menu-btn" id="btn3">Notifications</a>
+            <a href="#" class="account-settings__menu-btn" id="btn4">Security</a>
+          </div>
```
Comment (whamester) [inline pages/settings/index.html:54]: please rename these ids 🙏

**Comment by whamester** [comment]
Style comments:

Screenshot 2023-11-23 at 8 10 51 PM
 in order to achieve a better look on the select you can remove the default arrow with something like this (apply to the classes you think fit best not as generic as this example)
 
 ```
select {
  /* for Firefox */
  -moz-appearance: none;
  /* for Chrome */
  -webkit-appearance: none;
}

/* For IE10 */
select::-ms-expand {
  display: none;
}

```

Screenshot 2023-11-23 at 8 12 29 PM
Make the container of the image flex and allow flex wrap so the buttons will stack vertically

Notification icon
Current
Screenshot 2023-11-23 at 8 17 26 PM

Figma
Screenshot 2023-11-23 at 8 17 43 PM

https://www.figma.com/file/EFlGK158VL4Cphasesw2aS/UI-Design?type=design&node-id=595-19926&mode=design&t=xYDBd8TirdbYxjOW-0


Check this paragraph styles:
Current
Screenshot 2023-11-23 at 8 18 25 PM

Figma:
Screenshot 2023-11-23 at 8 18 39 PM

Delete Account Button - Icon
Current
Screenshot 2023-11-23 at 8 19 51 PM


Figma:
image

Spacing:
Current
Screenshot 2023-11-23 at 8 24 05 PM

Expected:
2.5rem between header and title

**Comment by whamester** [comment]
Issue:
After I removed my profile image I cannot upload a new one.

**Comment by whamester** [comment]
Waiting for design to confirm delete image behavior: https://langara-coding.slack.com/archives/C05R1JVS0UE/p1700799345851279

**Comment by ReinhardtBotha** [comment]
All the comments have been addressed in the subsequent commits.

**Review by ALFAROSO — APPROVED** [review]

**Pull request merged**